p4-16/spec/P4-16-spec.mdk

```diff
@@ -1661,6 +1661,24 @@ directions:
   See Section [#sec-invoke-actions] for further details.
 - Default parameter values are only allowed for 'in' or direction-less parameters;
   these values must evaluate to compile-time constants.
+- If a callable object (action, control, parser, extern method, etc.)
+  has a signature containing default values for parameters, and if the
+  parameters containing default values do *not*  all appear at the end of
+  the list of parameters, then the object can be invoked only by
+  using named arguments, to make the call easy to disambiguate.
```

Comment:
This wording is confusing I think.
Especially when the example below shows an example of where not using named arguments is still allowed.
I don't have better wording on this but we definitely should combine the example below even with the example under @optional section below this too. Or rather move the example of default arguments only to this section from that section too.